src/components/Search/SearchRouter/SearchRouterInput.tsx

```diff
@@ -37,6 +37,9 @@ type SearchRouterInputProps = {
     /** Whether the input should be focused */
     autoFocus?: boolean;
 
+    /** Whether the input caret should be hidden or not */
+    caretHidden?: TextInputProps['caretHidden'];
```

Comment:
Use `Pick` so we don't have to define the prop. Do it for other props that we are just forward e.g. autoFocus.

```ts
... & Pick<TextInputProps, 'autoFocus' | 'caretHidden'>
```